eventlog/README.md

```diff
@@ -112,9 +107,5 @@ long now = System.currentTimeMillis() * 1000 * 1000;
 SearchResponse resp = el.search("", now - 1000, now + 1000);
 ```
 
-Please refer to the javadocs for more information. The javadocs are not hosted
```

Comment:
Want to give a link to javadoc.io?

Reply:
the java doc is given above https://github.com/dedis/cothority/pull/1911/files#diff-3ff608f4768b8ab0247453c05725047fR76